src/modules/addressBook/hooks/useAddressBook.tsx

```diff
@@ -89,6 +91,7 @@ const useAddressBook = (isSingleIncluded: boolean = false) => {
       ];
       await listContactsPaginatedRequest.refetch();
       await listContactsRequest.refetch();
+      invalidateQueries();
       queryClient.invalidateQueries([...queryKeysToInvalidate, true]);
```

Comment:
why you dont add the query with this list on invalidate? 
`queryKeysToInvalidate `